addOns/automation/src/main/java/org/zaproxy/addon/automation/gui/NewPlanDialog.java

```diff
@@ -64,7 +65,12 @@ public class NewPlanDialog extends StandardFieldsDialog {
     private static final String REPORT_JOB_NAME = "report";
 
     private static final String[] BASELINE_PROFILE = {
-        "passiveScan-config", "spider", "spiderAjax", "passiveScan-wait", REPORT_JOB_NAME
+        "passiveScan-config",
+        "spider",
+        "spiderAjax",
+        "passiveScan-wait",
```

Comment:
Not for this PR, but it looks like we should create constants for all these.

Reply:
As long as they are not public.